### DISCUSSION TITLE
Charts are not necessarily 0-based on the Y axis

### DISCUSSION THREAD

**Discussion by chrisglein** [community/community#12075] [Projects and Issues]
I've been excited to have burndown charts off my new project board. It took me awhile to realize that the chart's Y axis doesn't start at 0.
![image](https://user-images.githubusercontent.com/26607885/155635515-a5a37553-60bf-4acc-8272-9a2040d053a8.png)

Now maybe that's the view you want... or maybe not. I found it very confusing and wanted the option for a view to anchor that Y axis to 0.

**Comment by randseay** [comment]
Not an answer but I was also disoriented when the y-axis was not zero-based. Would love to see this as a possibility (if not the default)!
